proposals/2025-02-24_secret_providers.md

```diff
@@ -0,0 +1,277 @@
+## Remote Secrets (Secret Providers)
+
+* **Owners:**
+  * Henrique Matulis (@hsmatulisgoogle)
```

Comment:
nit: username doesn't match (anymore? I think it was changed)

Reply:
Fixed, thanks!